com.yubico.YubiOath.json

```diff
@@ -0,0 +1,257 @@
+{
+    "app-id": "com.yubico.YubiOath",
+    "runtime": "org.kde.Platform",
+    "sdk": "org.kde.Sdk",
+    "runtime-version": "5.12",
+    "command": "yubioath-desktop",
+    "cleanup": [
+	"/include",
+	"/lib/pkgconfig"
+    ],
+    "finish-args": [
+        "--socket=x11",
+	"--socket=pcsc",
+	/* We need to be able to list usb devices, even when using CCID */
+	"--device=all"
+    ],
+    "modules": [
+	{
+	    "name": "libusb",
+	    "config-opts": ["--enable-udev=no"],
+	    "cleanup": [
+		"/lib/libusb-1.0.so"
+	    ],
+	    "sources": [
+		{
+                    "type": "archive",
+		    "url": "https://github.com/libusb/libusb/releases/download/v1.0.22/libusb-1.0.22.tar.bz2",
+		    "sha256": "75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157"
+		}
+	    ]
+	},
+	{
+	    "name": "pcsc-lite",
+	    "config-opts": [
+		"--disable-libudev",
+		"--disable-libsystemd",
+		"--without-systemdsystemunitdir"],
+	    "post-install": [
+		"rm /app/sbin/pcscd",
+		"rmdir /app/sbin || true"
+	    ],
+	    "cleanup": [
+		"/lib/libpcscspy.so",
+		"/lib/libpcsclite.so"
+	    ],
+	    "sources": [
+		{
+                    "type": "archive",
+                    "url": "https://github.com/LudovicRousseau/PCSC/archive/pcsc-1.8.24.tar.gz",
+		    "sha256": "6b079ff4fbcbc1a181aa66b8f184a07865e1cb55d8d9b8aaa26b595bb0088f51"
+		}
+	    ]
+	},
+	{
+	    "name": "pyotherside",
+            "buildsystem": "qmake",
+	    "sources": [
+		{
+                    "type": "archive",
+                    "url": "https://github.com/thp/pyotherside/archive/1.5.4.tar.gz",
+		    "sha256": "5f218778fc60377cee44b0edf6deaa6dd6aa84a109ebbb5b13d52cec31696b80"
+		},
+		{
+		    "type": "patch",
+		    "path": "pyotherside-install-path.patch"
+		}
+	    ]
+	},
+	{
+	    "name": "swig",
+	    "config-opts": [
+		"--without-alllang",
+		"--with-python3"
+	    ],
+	    "cleanup": [
+		"/**"
+	    ],
+	    "sources": [
+		{
+                    "type": "archive",
+                    "url": "http://download.sourceforge.net/swig/swig-3.0.12.tar.gz",
+		    "sha256": "7cf9f447ae7ed1c51722efc45e7f14418d15d7a1e143ac9f09a668999f4fc94d"
+		}
+	    ]
+	},
+	{
+            "name": "pyscard",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/LudovicRousseau/pyscard/archive/release-1.9.7.tar.gz",
```

Comment:
latest is [1.9.9](https://github.com/LudovicRousseau/pyscard/releases/tag/release-1.9.9)

Reply:
Fixed